### ISSUE TITLE
feat: Support more of the options that are available when calling flutter run

### ISSUE THREAD

**Issue by k-ane** [wolfenrain/fluttium#388]
Right now I can't run tests with the following which is blocking me in some cases:
- --web-port
- --dart-define-from-file

Would we be able to get them added as options in the fluttium.yaml as well as when running tests?

**Comment by mrgnhnt96** [comment]
Without `--dart-define-from-file`, using fluttium will be a bit cumbersome..

**Comment by wolfenrain** [comment]
Yes that would be quite useful indeed! Shouldn't be hard to add either as both the YAML config and the CLI are pretty straight forward in how they add parameters if anyone wants to try.

I am not sure when I'll get to it myself as I am trying to refocus on #293
